dev_guide/pod_autoscaling.adoc

```diff
@@ -215,6 +230,21 @@ spec:
 $ oc create -f hpa.yaml
 ----
 
+[IMPORTANT]
+====
+For memory-based autoscaling to work, memory usage must increase and decrease
+proportionally to the replica count. On average:
+
+* An increase in replica count must lead to an overall decrease in memory
+(working set) usage per-pod.
+* A decrease in replica count must lead to an overall increase in per-pod memory
+usage.
+
+Use the OpenShift dashboard to verify the memory behavior of your application
```

Comment:
What is the OpenShift dashboard? New in 3.9 and I am not aware? Should this be Kibana dashboard?

Reply:
s/dashboard/webconsole/